sdks/python/apache_beam/typehints/typed_pipeline_test.py

```diff
@@ -211,6 +211,31 @@ def process(self, element):
     assert_that(res_odd, equal_to([1, 3]), label='odd_check')
     p.run()
 
+  def test_typed_ptransform_fn_pre_hints(self):
+    # Test that type hints are propagated to the created PTransform.
+    # Decorator appears before type hints. This is the more common style.
+    @beam.ptransform_fn
+    @typehints.with_input_types(int)
```

Comment:
Example above reads slightly differently. It is:

    @ptransform_fn
    @beam.typehints.with_input_types(..)

Reply:
What exactly is different?
Do you mean `@beam.transform_fn` vs `@transform_fn`?

Reply:
I was thinking about `@beam.typehints` vs `typehints`. So typehints in beam.typehints here?

github reviews are difficult sometimes, with github not showing the whole file easily.

Reply:
Yeah, typehints is always beam.typehints. Python's native module is called typing.